spec/controllers/boards_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "rails_helper"
+
+RSpec.describe BoardsController, type: :controller do
+
+  before do
+    ApplicationController.any_instance.stub(:current_user).and_return(true)
+  end
+
+  describe "GET index" do
+    it "assigns @boards" do
+      board = Board.create
```

Comment:
tutaj zeby stworzyc let'em obiekt to robisz
`let!(:board) { create(:board) }` wykrzyknik wymaga stworzenia obiektu.
Jęsli jest let bez wykrzyknika to wtedy obiekt dopiero sie tworzy gdy jest wywoływany.